lmdeploy/cli/utils.py

```diff
@@ -354,3 +354,13 @@ def work_dir(parser):
             type=str,
             default='./work_dir',
             help='The working directory to save results')
+
+    @staticmethod
+    def cache_block_seq_len(parser):
+        """Add argument cache_block_seq_len to parser."""
+
+        return parser.add_argument(
+            '--cache-block-seq-len',
+            type=int,
+            default=64,
+            help='The length of the token sequence in a k/v block')
```

Comment:
Better add more help info， eg. it should be multipliers of 32/64?  and range?

Reply:
@RunningLeon @lvhan028 For Turbomind engine, it should be multipliers of 32 or 64 for different GPU compute capability version. For Pytorch engine, it maybe have different requirement. Do we need to specify the requirement here?

Reply:
More help message are added. @lvhan028 could you help review this PR?